dashboard/src/components/TreeListingPage/TreeListingPage.tsx

```diff
@@ -100,15 +100,20 @@ const TreeListingPage = ({ inputFilter }: ITreeListingPage): JSX.Element => {
 
   return data && data.length > 0 ? (
     <div className="flex flex-col gap-6">
-      <div className="flex flex-col items-end gap-4">
-        <TableInfo
-          startIndex={startIndex + 1}
-          endIndex={endIndex}
-          totalTrees={listItems.length}
-          itemsPerPage={ITEMS_PER_PAGE}
-          onClickBack={onClickGoBack}
-          onClickForward={onClickGoForward}
-        />
+      <div className="flex items-center justify-between gap-4">
+        <span className="text-left">
```

Comment:
NIT: Just a suggestion of style for this:
```suggestion
        <span className="text-left text-sm text-dimGray">
```
I also think it would be better to put a break line

| Previous | Now |
|--------|--------|
| <img width="730" alt="image" src="https://github.com/user-attachments/assets/207c0948-bf7c-4e03-9a7a-1eaca7aae919">|<img width="720" alt="image" src="https://github.com/user-attachments/assets/15777130-f866-4cf6-81c8-f917cfee46bb">|